src/interfaces/koa2.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Should this be `2018` since it's a new file?

Reply:
Yep good catch.